cmd/ceremony/main.go

```diff
@@ -22,6 +23,156 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+// issueLintCert issues a linting certificate from a given template certificate
+// signed by a given issuer and returns the certificate or an error. The public
+// key from the loaded certificate is checked by the GoodKey package.
+func issueLintCert(tbs, issuer *x509.Certificate, subjectPubKey crypto.PublicKey, signer crypto.Signer, skipLints []string) (*x509.Certificate, error) {
```

Comment:
nit: this both creates _and_ actually lints the lint cert. Perhaps indicate that in the name?